II.pass-tests/05-factorial/factorial.js

```diff
@@ -10,6 +10,21 @@
 
 function factorial(int) {
 
-}
+    if (int === 0||int === 1) {
+
+        return 1;
+    
+      } else {
+    
+        for(var i = int-1; i > 0; --i) { //always make sure to decrement the value BEFORE it's tacked onto the original as a product
```

Comment:
That's a good clarifying comment, it explains *why* the pre-decrement operator is being used. But I mentioned in class that if you think you need to explain something maybe the code isn't quite clear enough itself; could you think of another implementation that's more self-explanator?